lusSTR/repeat.py

```diff
@@ -49,18 +49,22 @@ def collapse_all_repeats(sequence, repeats):
     return collapsed_seq
 
 
-def split_by_n(sequence, n):
+def split_by_n(sequence, n, rev):
     '''Split a sequence into non-overlapping chunks of length n.'''
     while sequence:
-        yield sequence[:n]
-        sequence = sequence[n:]
+        if rev is False:
+            yield sequence[:n]
+            sequence = sequence[n:]
+        else:
+            yield sequence[-n:]
+            sequence = sequence[:-n]
 
 
-def get_blocks(sequence, n):
+def get_blocks(sequence, n, rev):
```

Comment:
Same here, a default value of True for rev might be a good option.

Reply:
I added a default value of False.